WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -420,44 +419,6 @@ public void onDetach() {
     }
 
     /*
-     * called by the adapter after posts have been loaded
-     */
-    @Override
-    public void onPostsLoaded(int postCount) {
-        if (!isAdded()) {
-            return;
-        }
-
-        if (postCount == 0 && !mIsFetchingPosts) {
-            if (NetworkUtils.isNetworkAvailable(getActivity())) {
-                updateEmptyView(EmptyViewMessageType.NO_CONTENT);
-            } else {
-                updateEmptyView(EmptyViewMessageType.NETWORK_ERROR);
-            }
-        } else if (postCount > 0) {
-            hideEmptyView();
-        }
-    }
-
-    /*
-     * called by the adapter to load more posts when the user scrolls towards the last post
-     */
-    @Override
-    public void onLoadMore() {
```

Comment:
How are we triggering the fetch for the next set of posts/pages if we remove this?